recipes/colorlog/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "2.6.1" %}
+
+package:
+  name: colorlog
+  version: {{ version }}
+
+source:
+  fn: colorlog-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/c/colorlog/colorlog-{{ version }}.tar.gz
+  md5: 712c85c4c44db0098bed09ed30a18a37
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - colorama  # [win]
+
+  run:
+    - python
+    - colorama  # [win]
+
+test:
+  imports:
+    - colorlog
+
+about:
+  home: https://github.com/borntyping/python-colorlog
+  license: MIT License
```

Comment:
You can change `MIT License` to just `MIT`. (The filed is named license :wink: already.)